boards/renesas/ek_ra8d1/ek_ra8d1.dts

```diff
@@ -74,6 +75,24 @@
 	status = "okay";
 };
 
+&pclka {
+	clk-src = <RA_CLOCK_SOURCE_PLL1P>;
+	clk-div = <RA_SYS_CLOCK_DIV_4>;
+	status = "okay";
+};
+
+&pclke {
+	clk-src = <RA_CLOCK_SOURCE_PLL1P>;
+	clk-div = <RA_SYS_CLOCK_DIV_2>;
+	status = "okay";
+};
```

Comment:
@thenguyenyf : I think that these clocks are same with clocks in [r7fa8d1xh.dtsi](https://github.com/renesas/zephyr/blob/support_renesas_ra8_canfd/dts/arm/renesas/ra/ra8/r7fa8d1xh.dtsi)

Reply:
Please also pay attention about 
https://github.com/zephyrproject-rtos/zephyr/pull/78365

Reply:
I reverted this change to use default config from the SoC devicetree